modules/gapi/include/opencv2/gapi/infer/bindings_ie.hpp

```diff
@@ -22,17 +22,23 @@ namespace ie {
 // This class can be marked as SIMPLE, because it's implemented as pimpl
 class GAPI_EXPORTS_W_SIMPLE PyParams {
 public:
-    PyParams() = default;
+    GAPI_WRAP PyParams() = default;
 
-    PyParams(const std::string &tag,
+    GAPI_WRAP PyParams(const std::string &tag,
              const std::string &model,
              const std::string &weights,
              const std::string &device);
 
-    PyParams(const std::string &tag,
+    GAPI_WRAP PyParams(const std::string &tag,
```

Comment:
Alignment broken, after adding GAPI_WRAP

Reply:
Done

Reply:
It is not done for the ctor above.
It makes sense to put "wrapping" modifiers on new line before standard declaration instead:

```.cpp
    GAPI_WRAP
    PyParams(...
```

for ctor and other methods / functions.

Reply:
Done